src/core/core.controller.js

```diff
@@ -476,10 +476,13 @@ class Chart {
 
 		me._updateLayout();
 
-		// Can only reset the new controllers after the scales have been updated
-		each(newControllers, (controller) => {
-			controller.reset();
-		});
+		// Only reset the conrtollers if we have animations
+		if (!animsDisabled) {
+			// Can only reset the new controllers after the scales have been updated
```

Comment:
Might be good to include why we reset the controllers if animations are enabled, i.e. it gives us the starting point for the animation